commands/contract/deploy.go

```diff
@@ -51,39 +75,69 @@ var deployCmd = &cobra.Command{
 			fmt.Println(err)
 			os.Exit(1)
 		}
-		modelFile, err := ContractCmd.PersistentFlags().GetString("modelfile")
+		version, err := ContractCmd.PersistentFlags().GetString("version")
 		if err != nil {
 			fmt.Println(err)
 			os.Exit(1)
 		}
-		deployer, err := GetDeployer(blockchain, modelFile)
+
+		switch strings.ToUpper(blockchain) {
+		case strings.ToUpper(config.HYPERLEDGER_FABRIC):
+			deployer := &fabc.Deployer{}
+			err := deployer.Deploy(
+				fabc.WithChaincodeID(id),
+				fabc.WithChaincodePath(path),
+				fabc.WithChaincodeVersion(version),
+				fabc.WithUserName(userName),
+				fabc.WithOrgName(orgName),
+				fabc.WithFabricConfig(networkConfig, networkOverride),
+			)
+			if err != nil {
+				fmt.Println(err)
+				os.Exit(1)
+			}
+		default:
+			fmt.Println("Unsupported blockchain for deploy:", blockchain)
+		}
+	},
+}
+
+var instantiateCmd = &cobra.Command{
```

Comment:
Also here we should think if this "instantiate" concept will be available for all blockchains supported and how does that affect the command structure

Reply:
ditto